src/components/worker.js

```diff
@@ -1,32 +1,33 @@
+import {presets} from "../util/types";
```

Comment:
why is this file a .js file now?

Reply:
it's always been a .js file, but good catch.  i'll change it to .tsx